README.md

```diff
@@ -111,6 +111,11 @@ class User {
 * Node 8.10+
 * mongoose ^5.9.2
 * `emitDecoratorMetadata` and `experimentalDecorators` must be enabled in `tsconfig.json`
+  * if you use Babel (e.g. with Jest), it will need the corresponding plugins to complement the [typescript preset](https://babeljs.io/docs/en/babel-preset-typescript):
```

Comment:
please fix the extra 2 space indentation

Reply:
If I removed those, the block didn't get formatted as code by GitHub. I'll see about making it a code block.